Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3842,10 +3842,12 @@ def gen2021HiMix(fragment,howMuch):
     elif '2018' in ds:
         PUDataSets[ds]={'-n':10,'--pileup':'AVE_50_BX_25ns','--pileup_input':'das:/RelValMinBias_13/%s/GEN-SIM'%(name,)}
     elif '2021' in ds:
-        if 'FS' not in ds:
+        if 'FS' in ds:
+            PUDataSets[ds]={'-n':10,'--pileup':'Run3_Flat55To75_PoissonOOTPU','--pileup_input':'das:/RelValMinBias_14TeV/%s/GEN-SIM-RECO'%(name,)}
+        elif 'Design' in ds:
             PUDataSets[ds]={'-n':10,'--pileup':'Run3_Flat55To75_PoissonOOTPU','--pileup_input':'das:/RelValMinBias_14TeV/%s/GEN-SIM'%(name,)}
         else:
-            PUDataSets[ds]={'-n':10,'--pileup':'Run3_Flat55To75_PoissonOOTPU','--pileup_input':'das:/RelValMinBias_14TeV/%s/GEN-SIM-RECO'%(name,)}
+            PUDataSets[ds]={'-n':10,'--pileup':'Run3_Flat55To75_PoissonOOTPU','--pileup_input':'das:/RelValMinBias_13p6TeV/%s/GEN-SIM'%(name,)}
```

Comment:
revert this please (3845-3850). (line 3810 should stay the same iiuc)

Reply:
thanks Bugra! done in [48eaa12](https://github.com/cms-sw/cmssw/pull/41123/commits/48eaa12b3188efc51681f77f216fdfa8446a22a4)
I'll restart the tests since the new GS samples seem ready!